frontend/src/contexts/auth.context.tsx

```diff
@@ -99,12 +101,17 @@ export const AuthProvider = ({ children }: AuthProviderProps): JSX.Element => {
     setUser(user);
   };
   const isAuthenticated = !!user;
+  const { subscribe } = useBroadcastChannel();
```

Comment:
```suggestion
  useSubscribeBroadcastChannel("logout", () => {
      router.reload();
    });
```
Let's create a new hook ;)